### ISSUE TITLE
Support to add private images through CasaOS

### ISSUE THREAD

**Issue by Jerakin** [IceWhaleTech/CasaOS#908]
Personally I would be fine with logging in to Docker through ssh but it might be more "complete" if there is an ability to login through the CasaOS to get access to the private images.

**Comment by jerrykuku** [comment]
There is a terminal panel for the docker container. Did you mean that?
![image](https://user-images.githubusercontent.com/9485680/223155889-c60962da-67a9-4c53-8730-9ca93e20536e.png)

**Comment by Jerakin** [comment]
No, I would like to be able to install a private image through the normal interface

![2023-03-07 23_37_30-CasaOS — Mozilla Firefox](https://user-images.githubusercontent.com/5273682/223569801-1783664e-b3cd-40c9-a1db-86b7355db5bd.png)
